3rdParty/iresearch/core/shared.hpp

```diff
@@ -206,6 +206,15 @@
   #define MSVC2017_ONLY(...)
 #endif
 
+// hook for MSVC2019-only code
+#if defined(_MSC_VER) \
+    && (_MSC_VER == 1920 \
+        || false )
```

Comment:
why `|| false` here?

Reply:
It is a placeholder but can be safely deleted.

Reply:
Please delete it then.